### PR TITLE
Add Hunger and Thirst to constant modded change

### PR DESCRIPTION
**Pull request by Diakon5**
I added Hunger and Thirst based on the implementation in stress. I am uncertain how to implement it for panic, sleep and fatigue, due to those stats being adjusted in a different way(as they both increase and decrease separately, instead of being concatenated like stress). Perhaps having different tables for a constant increase and a constant decrease.

### CODE REVIEW

File: Contents/mods/StatsAPI/media/lua/client/StatsAPI/StatsAPI.lua [hunk 1]
```diff
@@ -124,19 +124,38 @@ StatsAPI.addStressChangeDaily = function(sourceName, dailyChange)
     StatsAPI.Stress.modChanges[sourceName] = dailyChange / 86400 / 100
 end
 
----Adds a constant change to stress
+---Same as above, but calculated over an hour for higher precision
```
Comment (demiurgeQuantified) [inline Contents/mods/StatsAPI/media/lua/client/StatsAPI/StatsAPI.lua]: These comments are intended to be read in the mouse-over tooltips, so 'same as above' should be avoided

Comment (Diakon5) [reply Contents/mods/StatsAPI/media/lua/client/StatsAPI/StatsAPI.lua]: Fixed with current PR for stress

**Comment by Diakon5** [comment]
Made some updates, now the code for stress, hunger and thirst theoretically accepts functions

**Comment by demiurgeQuantified** [comment]
ideally the stat change functions wouldn't replace the number stat change api calls, otherwise it makes what should be straightforward 'add x stress a day' calls somewhat complex

**Comment by Diakon5** [comment]
Bad coding time: Have it be a singular function that either applies it or calls a function?

I think I'll code it normally

**Comment by demiurgeQuantified** [comment]
i kind of scuffed up the git log rebasing things, but it seems fine

**Pull request merged**